jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -1,14 +1,17 @@
 from traitlets import Bool, List, Unicode, observe
 import regions
+import numpy as np

```

Comment:
Sometimes I like `isort` way of things but only sometimes.

```suggestion
import numpy as np
import regions
```